### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by mickeyvip** [juancabrera/gulp-traceur-cmdline#10]
I have `traceur` v0.0.79 installed locally and globally, using the example form the blog (`traceur` path is specified):

```javascript
gulp.task('default',function() {
  gulp.src("./src/todolist.module.js")
    .pipe(gulpTraceurCmdline('./node_modules/traceur/traceur', {
      modules : 'inline',
      out     : './out/todolist.module.js',
      debug   : false
    }))
});
```
or

```javascript
gulp.task('default',function() {
  gulp.src("./src/todolist.module.js")
    .pipe(gulpTraceurCmdline('~/npm/bin/traceur', {
      modules : 'inline',
      out     : './out/todolist.module.js',
      debug   : false
    }))
});
```

I get:
```bash
[19:58:23] Using gulpfile ~/dev/nodejs/ToDoListES6/gulpfile.js
[19:58:23] Starting 'default'...
[19:58:23] Finished 'default' after 4.85 ms
[19:58:23] 
       *** Error : "out" not provided!  ***

events.js:85
      throw er; // Unhandled 'error' event
            ^
TypeError: Object.keys called on non-object
    at keys (native)
    at Function.keys (/Users/mickey/dev/nodejs/ToDoListES6/node_modules/traceur/bin/traceur.js:151:29)
    at build_commandline_opts (/Users/mickey/dev/nodejs/ToDoListES6/node_modules/gulp-traceur-cmdline/lib/traceur_command_builder.js:50:36)
    at build_commandline (/Users/mickey/dev/nodejs/ToDoListES6/node_modules/gulp-traceur-cmdline/lib/traceur_command_builder.js:8:30)
    at /Users/mickey/dev/nodejs/ToDoListES6/node_modules/gulp-traceur-cmdline/index.js:27:19
    at wrappedMapper (/Users/mickey/dev/nodejs/ToDoListES6/node_modules/gulp-traceur-cmdline/node_modules/map-stream/index.js:84:19)
    at Stream.stream.write (/Users/mickey/dev/nodejs/ToDoListES6/node_modules/gulp-traceur-cmdline/node_modules/map-stream/index.js:96:21)
    at write (/Users/mickey/dev/nodejs/ToDoListES6/node_modules/gulp/node_modules/vinyl-fs/node_modules/through2/node_modules/readable-stream/lib/_stream_readable.js:623:24)
    at flow (/Users/mickey/dev/nodejs/ToDoListES6/node_modules/gulp/node_modules/vinyl-fs/node_modules/through2/node_modules/readable-stream/lib/_stream_readable.js:632:7)
    at DestroyableTransform.pipeOnReadable (/Users/mickey/dev/nodejs/ToDoListES6/node_modules/gulp/node_modules/vinyl-fs/node_modules/through2/node_modules/readable-stream/lib/_stream_readable.js:664:5)
```

Using the example from the GitHub:

```javascript
gulp.task('default',function() {
  gulp.src("./src/todolist.module.js")
    .pipe(gulpTraceurCmdline({
      modules : 'inline',
      out     : './out/todolist.module.js',
      debug   : false
    }))
});
```

I get

```bash
[20:03:42] Using gulpfile ~/dev/nodejs/ToDoListES6/gulpfile.js
[20:03:42] Starting 'default'...
[20:03:42] Finished 'default' after 4.74 ms
events.js:85
      throw er; // Unhandled 'error' event
            ^
Error: Command failed: /bin/sh -c ./node_modules/gulp-traceur-cmdline/node_modules/traceur/traceur --modules=inline --out=./out/todolist.module.js  /Users/mickey/dev/nodejs/ToDoListES6/src/todolist.module.js
/bin/sh: ./node_modules/gulp-traceur-cmdline/node_modules/traceur/traceur: No such file or directory

    at ChildProcess.exithandler (child_process.js:732:12)
    at ChildProcess.emit (events.js:110:17)
    at maybeClose (child_process.js:1000:16)
    at Socket.<anonymous> (child_process.js:1168:11)
    at Socket.emit (events.js:107:17)
    at Pipe.close (net.js:461:12)
```

and indeed `traceur` was not installed there

**Comment by juancabrera** [comment]
So this get fixed with #11, right ?

**Comment by mickeyvip** [comment]
yes
